common/db_test.go

```diff
@@ -118,6 +125,47 @@ func profilesMatch(j1 []ffmpeg.VideoProfile, j2 []ffmpeg.VideoProfile) bool {
 	return true
 }
 
+func TestDBOrchestrators(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+
+	orch := NewStubOrch("127.0.0.1:8936", "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe")
+	err = dbh.UpdateOrchs(orch)
+
+	if err != nil {
+		t.Error("Unexpected error in update orchs: ", err)
+	}
+
+	orchs, err := dbh.SelectOrchs()
+	if err != nil || len(orchs) != 1 || orchs[0].ServiceURI != orch.ServiceURI {
+		t.Error("Unexpected error in update orchs: ", err)
+	}
+
+	orchUpdate := NewStubOrch("127.0.0.1:8937", "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe")
```

Comment:
Does the ETH address need to be this specific value? - if not I personally like the idea of using random values, primarily because it clarifies to the test reader that we don't really care what value we use here ([see example](https://github.com/livepeer/go-livepeer/pull/655/files#diff-c20584ae20b069ed08dc6769a1c19e77R803)).

wdyt?

Reply:
I like the improvement - you're right in that it does not need to be a specific value